specification/sql/resource-manager/Microsoft.Sql/preview/2017-10-01-preview/databases.json

```diff
@@ -801,8 +812,8 @@
       ],
       "properties": {
         "sku": {
-          "$ref": "../../../common/v1/types.json#/definitions/Sku",
-          "description": "The name and tier of the SKU."
+          "$ref": "#/definitions/Sku",
```

Comment:
actually. Here is an error. The referenece is wrong. It should reference ../../../common/v1/types.json#/definitions/Sku